.gitignore

```diff
@@ -11,3 +11,4 @@ transaction.log
 .idea
 *.iml
 
+/src/test/java/org/ccci/util/mail/MailProperties.java
```

Comment:
Will be removed before deploy, both this line and the MailProperties file.
The file contains information gathered from secrets to send emails using ERT email credentials which should not end up in github

Reply:
I don't see any `MailProperties.java` file.
Note that you should never push secrets to github, regardless of whether or not you remove before deploy.

Reply:
You don't see `MailProperties.java` since it is in the gitignore.
The comment is intended to convey that i know the data is supposed to be secure, as such i am keeping it locally for testing pending any required changes before deploy but not pushing to GH.

Reply:
Ah, ok. Understood.